web/src/games/mastermind/media/circle.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!-- Font Awesome Free 5.15.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) --><path d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8z"/></svg>
```

Comment:
Importing svgs should work, but FYI another option is to treat them as react components as we do in seabattle:
https://github.com/freeboardgames/FreeBoardGames.org/blob/master/web/src/games/seabattle/media/SvgHit.tsx

The advantage is that would allow you to use only one <svg> and hence having a single viewport and etc... Maybe it is useful for you, maybe not :)